menas/src/main/scala/za/co/absa/enceladus/menas/repositories/VersionedMongoRepository.scala

```diff
@@ -132,6 +132,20 @@ abstract class VersionedMongoRepository[C <: VersionedModel](mongoDb: MongoDatab
       set("userDisabled", username))).toFuture()
   }
 
+  def setLockState(name: String, isLocked: Boolean, username: String,
```

Comment:
Not a fan of methods with boolean params, when it could be nicely 2 method `lock` and `unlock`, but I am not sure of our stance of that as a group.